client/src/components/Navbar/Navbar.tsx

```diff
@@ -113,82 +123,84 @@ const Navbar: React.FC = () => {
+        return !loggedInUser && <MenuItem key={menu.resource || index} index={index} {...menu} />;
       }
     });
   };
 
   return (
-    <Grid
-      className={clsx(classes.navbar, location.pathname === '/' && classes.transparentNavbar)}
-      justifyContent="space-between"
-      alignItems="center"
-      container
-    >
-      <Grid xs={4} md={6} item>
-        <img className={classes.navbarLogo} src={lovingSitterLogo} />
-      </Grid>
-      <Grid xs={8} md={6} item>
-        <Grid container alignItems="center" gap={2} justifyContent="flex-end">
-          {renderMenuItems()}
-          {loggedInUser && (
-            <Grid xs={2} item>
-              <>
-                <IconButton
-                  size="large"
-                  aria-label="account profile picture"
-                  aria-controls="menu-navbar"
-                  arais-haspopup="true"
-                  onClick={handleMenuOpen}
-                  color="inherit"
-                >
-                  <img style={{ width: 50 }} src={`https://robohash.org/${loggedInUser.email}`} />
-                </IconButton>
-                <Menu
-                  id="menu-appbar"
-                  anchorEl={anchorEl}
-                  anchorOrigin={{
-                    vertical: 'bottom',
-                    horizontal: 'right',
-                  }}
-                  keepMounted
-                  transformOrigin={{
-                    vertical: 'top',
-                    horizontal: 'right',
-                  }}
-                  open={open}
-                  onClose={handleClose}
-                >
-                  <DropdownMenuItem component={NavLink} to="/profile/settings" onClick={handleClose}>
-                    <ListItemIcon>
-                      <Settings fontSize="small" />
-                    </ListItemIcon>
-                    <ListItemText>Settings</ListItemText>
-                  </DropdownMenuItem>
-                  <DropdownMenuItem onClick={handleClose}>
-                    <ListItemIcon>
-                      <Person fontSize="small" />
-                    </ListItemIcon>
-                    <ListItemText>Profile</ListItemText>
-                  </DropdownMenuItem>
-                  <Divider />
-                  <DropdownMenuItem onClick={handleLogout}>
-                    <ListItemIcon>
-                      <Logout fontSize="small" />
-                    </ListItemIcon>
-                    <ListItemText>Logout</ListItemText>
-                  </DropdownMenuItem>
-                </Menu>
-              </>
-            </Grid>
-          )}
+    <UnreadNotifications>
+      <Grid
+        className={clsx(classes.navbar, location.pathname === '/' && classes.transparentNavbar)}
+        justifyContent="space-between"
+        alignItems="center"
+        container
+      >
+        <Grid xs={4} md={6} item>
+          <img className={classes.navbarLogo} src={lovingSitterLogo} />
```

Comment:
This is the navbar logo img, it only says I changed it because I wrapped the entire component with the "UnreadNotifications" wrapper. I will add an alt attribute to this, but note, this was not apart of the issue which is why I did not make changes to the already existing items within the Navbar. Only added components necessary to the issue.